messages/actions/steering_angle.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
I agree with this, and, looking at the schema definition, this seems to be more or less the case since this message spec is clearly (?) aimed for a rudder command?

Another thing to discuss here is if (an in that case how?) there should be a difference between a rudder command and an observation of a rudder command? In a more general sense, how do we differentiate between a command sent as part of a remote control setup(shore center -> vessel) and an observation of a rudder command sent as  part of a monitoring setup (vessel -> "anyone")?